### PR TITLE
add pisces pcb

### PR DESCRIPTION
**Pull request by khchen2004**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kwstudio/pisces/readme.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Pisces
+
+![Pisces](https://i.imgur.com/fqEBVD8.jpeg)
```
Comment (dunk2k) [inline keyboards/kwstudio/pisces/readme.md:3]:
```suggestion
![Pisces](https://i.imgur.com/fqEBVD8h.jpeg)
```

File: keyboards/kwstudio/pisces/info.json [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+                {"matrix":[5,13], "x":12.5, "y":5.25}, 
+                {"matrix":[5,14], "x":13.5, "y":5.25, "w":1.5}, 
+                {"matrix":[5,15], "x":15.25, "y":5.25}, 
+                {"matrix":[5,16], "x":16.25, "y":5.25}, 
+                {"matrix":[5,17], "x":17.25, "y":5.25}
+            ]
+        }
+    }
+}
+
```
Comment (dunk2k) [inline keyboards/kwstudio/pisces/info.json:128]:
```suggestion

```

File: keyboards/kwstudio/pisces/readme.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+
+* Keyboard Maintainer: [khchen](https://github.com/khchen2004)
+* Hardware Supported: Pisces PCBs, atmega32u4
```
Comment (dunk2k) [inline keyboards/kwstudio/pisces/readme.md:7]:
```suggestion
* Hardware Supported: Pisces PCB
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/kwstudio/pisces/info.json [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+                {"matrix":[5,11], "x":11, "y":5.25, "w":1.5}, 
+                {"matrix":[5,13], "x":12.5, "y":5.25}, 
+                {"matrix":[5,14], "x":13.5, "y":5.25, "w":1.5}, 
+                {"matrix":[5,15], "x":15.25, "y":5.25}, 
+                {"matrix":[5,16], "x":16.25, "y":5.25}, 
+                {"matrix":[5,17], "x":17.25, "y":5.25}
+            ]
+        }
+    }
+}
```
Comment (dunk2k) [inline keyboards/kwstudio/pisces/info.json:127]:
```suggestion
}

```

File: keyboards/kwstudio/pisces/readme.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+# Pisces
+
+![Pisces](https://i.imgur.com/fqEBVD8h.jpeg)
+
```
Comment (dunk2k) [inline keyboards/kwstudio/pisces/readme.md:4]:
```suggestion
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

**Review by dunk2k — APPROVED** [review]

**Comment by khchen2004** [comment]
@dunk2k updated

File: keyboards/kwstudio/pisces/info.json [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+    "manufacturer": "kws",
+    "maintainer": "kws",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": false,
```
Comment (waffle87) [inline keyboards/kwstudio/pisces/info.json:12]: This is preferred to be left enabled if possible.
```suggestion
        "mousekey": true,
```

**Review by waffle87 — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

For future reference, we recommend against committing to your `master` branch as you've done here, because pull requests from modified `master` branches can make it more difficult to keep your QMK fork updated. It is highly recommended for QMK development – regardless of what is being done or where – to keep your master updated, but **NEVER** commit to it. Instead, do all your changes in a branch (branches are basically free in Git) and issue PRs from your branches when you're developing.

There are instructions on how to keep your fork updated here:

[**Best Practices: Your Fork's Master: Update Often, Commit Never**](https://docs.qmk.fm/#/newbs_git_using_your_master_branch)

[Fixing Your Branch](https://docs.qmk.fm/#/newbs_git_resynchronize_a_branch) will walk you through fixing up your `master` branch moving forward. If you need any help with this just ask.

Thanks for contributing!

**Pull request merged**